Assets/MRTK/Providers/WindowsMixedReality/Shared/Definitions/HPMotionControllerDefinition.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    public class HPMotionControllerDefinition
+    {
+        public HPMotionControllerDefinition(IMixedRealityInputSource source, Handedness handedness)
+        {
+            inputSource = source;
+            this.handedness = handedness;
+        }
+
+        protected readonly IMixedRealityInputSource inputSource;
+        protected readonly Handedness handedness;
+
+        /// <inheritdoc />
+        public MixedRealityInteractionMapping[] DefaultLeftHandedInteractions => new[]
+        {
+            new MixedRealityInteractionMapping(0, "Spatial Pointer", AxisType.SixDof, DeviceInputType.SpatialPointer),
+            new MixedRealityInteractionMapping(1, "Spatial Grip", AxisType.SixDof, DeviceInputType.SpatialGrip),
+            new MixedRealityInteractionMapping(2, "Grip Position", AxisType.SingleAxis, DeviceInputType.Grip),
+            new MixedRealityInteractionMapping(3, "Grip Touch", AxisType.Digital, DeviceInputType.GripTouch),
```

Comment:
This is maybe an interesting discussion point, but even though `DeviceInputType.GripTouch` has been defined for a while, it doesn't look like ANY controller actually implements it. Do we want to add it new here? Or just leave it out for consistency?

Reply:
I thought it was more of a catch all "if the code we are using anticipates handling a grip touch event, we want the controller to be firing it" kind of situation. I noticed that the GenericXRSDKController has a case of it in it's UpdateButton Data

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/4e1c3976b0f8403098bf6a291044c4eb873b5bd2/Assets/MRTK/Providers/XRSDK/Controllers/GenericXRSDKController.cs#L148-L151

Reply:
Yeah, interestingly that line in GenericXRSDKController appears to be the only reference to it outside the enum that defines it :/

https://github.com/microsoft/MixedRealityToolkit-Unity/search?q=DeviceInputType.GripTouch

Reply:
I'm all for defining it, though it's an odd case where

1. This is an interaction that isn't coming directly from the controller, but instead we're sorta simulating with `pressedValue > 0`
1. No other controller has it, even though they all could!